sdk/appconfiguration/app-configuration/samples-dev/listLabels.ts

```diff
@@ -63,9 +63,9 @@ export async function main() {
     console.log(`  Found label for development: ${label.name}`);
   }
 
-  ////////////////////////////////////////////////////////
-  ///////////////  Example for .byPage()  ////////////////
-  ////////////////////////////////////////////////////////
+  /// /////////////////////////////////////////////////////
```

Comment:
I don't think this should be intentional here.

Reply:
yeah that's the result of formatting.  We should probably rewrite it to block style to prevent this from happening